docs/api/app.md

```diff
@@ -565,9 +565,11 @@ the active app. On Windows, focuses on the application's first window.
 
 Hides all application windows without minimizing them.
 
-### `app.show()` _macOS_
+### `app.show(noActivate)` _macOS_
```

Comment:
Can we make this an options object with a `shouldActivate` key.

`app.show(false)` gives your average reader no idea as to what is about to happen or what the `false` means